### PR TITLE
new text login

### PR DESCRIPTION
**Pull request by zakiteravintech08**

### CODE REVIEW

File: src/featured/Login/index.tsx [hunk 1]
```diff
@@ -28,7 +28,7 @@ const Login = ({navigation}: any) => {
     <View style={styles.page}>
       <View style={styles.wrapper}>
         <Image source={Avatar} style={styles.avatar} />
-        <Text style={styles.welcome}>Welcome Back</Text>
+        <Text style={styles.welcome}>Welcome Back disini</Text>
```
Comment (iqbal-afrianto) [inline src/featured/Login/index.tsx]: Review

Comment (iqbal-afrianto) [reply src/featured/Login/index.tsx:31]: Mangsut?

**Review by iqbal-afrianto — CHANGES_REQUESTED** [review]

File: src/featured/Login/index.tsx (hunk 1 above)
Comment (iqbal-afrianto) [reply src/featured/Login/index.tsx]: Tolong ubah jadi 'Welcome Book'

Comment (zakiteravintech08) [reply src/featured/Login/index.tsx]: tolong di check lagi ya

**Review by iqbal-afrianto — APPROVED** [review]

**Pull request merged**